microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -229,4 +229,18 @@ Adds OLM image references in dedicated file::
 
 Issued: 2024-03-20
 
-{product-title} release 4.15.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1257[RHBA-2024:1257] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1255[RHSA-2024:1255] advisory.
\ No newline at end of file
+{product-title} release 4.15.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1257[RHBA-2024:1257] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1255[RHSA-2024:1255] advisory.
+
+[id="microshift-4-15-4-dp"]
+=== RHBA-2024:1451 - {microshift-short} 4.15.4 bug fix and enhancement update advisory
+
+Issued: 2024-03-26
+
+{product-title} release 4.15.4 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1451[RHBA-2024:1451] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1449[RHSA-2024:1449] advisory.
+
+[discrete]
+[id="microshift-4-15-4-enhancements"]
+==== Enhancements
+
+Change in default value::
+* In this release, the default value for the `securityContextConfig` parameter is changed from "restricted" to "baseline". (link:https://issues.redhat.com/browse/OCPBUGS-30034[OCPBUGS-30034])
```

Comment:
@aravipra I do not think we use word `securityContextConfig` here because it could make the user to change 
```yaml
spec:
  grpcPodConfig:
    securityContextConfig: restricted
```
to
```yaml
spec:
  grpcPodConfig:
    securityContextConfig: baseline
```

but actually no.

after changing `pod-security.kubernetes.io/enforce: restricted` to `pod-security.kubernetes.io/enforce: baseline` for openshift-marketplace in microshift with https://issues.redhat.com/browse/OCPBUGS-30034, when creating catalog source in openshift-marketplace, could keep
```yaml
spec:
  grpcPodConfig:
    securityContextConfig: restricted
```
or remove this part to take default value legacy.

if catalog source is create in other ns which enforce is restricted, still keep it.
```yaml
spec:
  grpcPodConfig:
    securityContextConfig: restricted
```

so, with https://issues.redhat.com/browse/OCPBUGS-30034, I suggest to change the word to
```console
In this release, if you create catalog source in openshift-marketplace, do not need to set securityContextConfig. if you create catalog source in other ns, still need to set securityContextConfig to restricted
```

by the way, securityContextConfig in catalogsource only support restricted or legacy.

cc @kevinrizza @jianzhangbjz @pmtk

Reply:
done